tfx_bsl/beam/run_inference_arrow.py

```diff
@@ -117,17 +116,25 @@ def RunInferenceImpl(  # pylint: disable=invalid-name
   """
   logging.info('RunInference on model: %s', inference_spec_type)
 
-  batched_examples = examples | 'BatchExamples' >> beam.BatchElements()
+  # TODO (Maxine): uncomment this once we change the api to take input 
+  # Union[tf.train.Example, tf.train.SequenceExample]
+  # data_type = _get_data_type(examples)
+
+  data_type = DataType.EXAMPLE
```

Comment:
Can this be auto detected and error out for classify, regress and multi-inference if it's not tf.Example?